lib/src/observable_value.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+/// A single variable with a stream emitting the variable's changing values.
+///
+/// Contains a single [value] which can be read and written as any other
+/// variable.
+///
+/// Whenever the value is set, the new value is emitted on the
+/// [updates] stream.
+/// Further, any new listener on the [updates] stream is initially sent the
+/// current [value].
+///
+/// Consider using immutable types for [value] since listeners will not
+/// be notified if the internal state of [value] changes, only if
+/// the [value] variable is written to.
+///
```

Comment:
```suggestion
/// This is particularly useful for a robust redux-like state management approach.
/// 
```

Reply:
That would definitely need a link. Until you mentioned it, I had never heard of "redux state mangement".

Reply:
> why we only support that one thing

Here we only mention a particularly helpful use case, but in no way we want to suggest that we don't support other use cases.

A nice link might be https://ngrx.io/guide/store/why

Reply:
I meant: Why do we not support the rest of the reactive operations from Rx? Why just the "BehaviorSubject"?

It's a nice primitive, but RxDart already has it.
So, why is `dart:async` providing this particular building block by itself, and not any of the other related ones?

Reply:
Not sure I'm good with writing something this specific. If you already know, you don't need the text saying it, and if you don't, it's *far* too specific to put in front of the average user.